tests/storage/restore.go

```diff
@@ -1229,7 +1229,10 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 
 				originalPVCName := pvc.Name
 
-				vmi = tests.NewRandomVMIWithPVCAndUserData(pvc.Name, bashHelloScript)
+				vmi = tests.NewRandomVMI(
```

Comment:
as mentioned elsewhere, I'd rather see you drop the usage of `tests` completely, by inlining what `NewRandomVMI` does.